File: benchmark/results/java-multilang/run-001/b-baseline/output/com/metaengine/demo/ordering/services/OrderService.java
package com.metaengine.demo.ordering.services;

import java.util.List;

import com.metaengine.demo.ordering.aggregates.Order;

/** OrderService for the ordering domain. */
public class OrderService {

    /** Creates a new Order from the partial input. */
    public Order create(Order input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Finds an Order by id. May return null. */
    public Order findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Lists Orders up to the given limit. */
    public List<Order> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Deletes the Order with the given id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/ts-monolith-sonnet-inline/run-001/b-baseline/output/src/orders/services/OrderService.ts
import { Id } from '../../shared/value-objects/Id';
import { Order } from '../aggregates/Order';

/** Manages customer orders. */
export class OrderService {
  create(input: Partial<Order>): Order {
    void input;
    throw new Error('not implemented');
  }

  findById(id: Id): Order | null {
    void id;
    throw new Error('not implemented');
  }

  list(limit: number): Order[] {
    void limit;
    throw new Error('not implemented');
  }

  delete(id: Id): void {
    void id;
    throw new Error('not implemented');
  }
}
